### PR TITLE
Initial Readme Creation

### PR DESCRIPTION
**Pull request by addamh**
I wanted to put this into a github project and have applicants fork the project as opposed to having a google doc they reference. I did a bit of rewording/rewriting @ckinsey hope you don't mind. 😝  Would also love your thoughts too @tycoles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,86 @@
...
+
+`The Project` will explore your capability as a Python developer using our preferred tooling. We don't expect all applicants to have proficiency in all of the technologies specified in this document.  Lofty Labs Consultants often work independently with only a small amount of direct supervision and often work directly on projects built by other engineering teams as well. Sometimes this introduces new systems to us that we haven't used before. This project is meant to expose you to the same situation wherein maybe you haven't used all of the technologies or tooling that we have specified in the requirements. Completion of `The Project` will show that even if you don't have experience with all of the specifications listed, you're a dedicated and passionate engineer that is able to pick up new systems and technologies at an at least basic level. The ability to adapt to a new and possibly unknown technology is a core competency of all Lofty Engineering Consultants.
```
Comment (ckinsey) [inline README.md]: > Lofty Labs Consultants often work independently with only a small amount of direct supervision and often work directly on projects built by other engineering teams as well.

Lofty Labs Consultants often work independently with only a small amount of direct supervision and, frequently, on projects built by other engineering teams.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,86 @@
...
+
+## Getting Help
+
+While working on this project, you will have access to members of the Lofty Labs consulting team via Slack. Contact us at `jobs@hirelofty.com` and let us know you'd like to be a part of the project and we'll add you to our slack channel named `#the-academy`. If you get stuck, or even just want to discuss the project in general, you are encouraged to ask questions and get help from the Lofty Engineering team as well as the other potential consultants working on the same project as you. A key component of the value we add to our client's projects as a consultantcy is our ability to communicate across many disparate teams and utilize everyone’s particular areas of expertise. Our clients don't hire us because we know every solution off-hand; our client's hire Lofty Labs because we know how to solve problems. Solving our client's problems includes research, continuing education, and cross-team knowledge sharing. What this means for you as a potential consultant working on `The Project` is that we don't mind you asking questions throughout your progress in slack. We don't care that you ask questions. We care that you ask *the right* questions.
```
Comment (ckinsey) [inline README.md:11]: `jobs@hirelofty.com`

need to get that set up.  I'll create it as a distribution list.  Can I put you on that, @addamh?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,86 @@
...
+
+## Description
+
+`The Project` requires you to build an analytics dashboard using Twitter data. The application should show usage information for a particular hashtag of your choosing and it should periodically retrieve data from Twitter’s API and store the results. The application should be interactive and highly performant. Since user's will be interacting with data cached in your application as opposed to pulled on request from Twitter, the application should be very performant and allow for smooth interactivity with the user.
```
Comment (ckinsey) [inline README.md]: > user's 

users

Comment (ckinsey) [inline README.md]: >  The application should be interactive and highly performant.

Can probably kill this sentence altogether.  Redundant with the one that follows.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,86 @@
...
+
+### Get Started!
+
+Good luck, and be sure to check the resources section below for further reading materials!
+
+### Resources
+
+In addition to the Lofty Labs Slack channel, here are some good places to start when researching the various tools needed to build this application.
+
+* [Very good beginner's Python reference](http://learnpythonthehardway.org/book/)
```
Comment (ckinsey) [inline README.md:75]: We might spend a little time digging up additional resources too.

**Comment by ckinsey** [comment]
Just a few comments on wording, but overall this looks good to me.

File: README.md (hunk 2 above)
Comment (addamh) [inline README.md:11]: yep i was just thinking how we could get people in the slack room and thought this might be quickest.

File: README.md (hunk 4 above)
Comment (addamh) [inline README.md:75]: @tycoles any resources you'd like to add that you've found are good for beginners?

**Comment by ckinsey** [comment]
Went ahead and moiged this so we could get it on the website in advance of PyCon.

**Pull request merged**